### ISSUE TITLE
App stuck on "UPDATE_ROLLBACK_FAILED state and can not be updated" after amplify update

### ISSUE THREAD

**Issue by luiskcs89** [aws-amplify/amplify-cli#9510]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### How did you install the Amplify CLI?

npm

### If applicable, what version of Node.js are you using?

v12.18.0

### Amplify CLI Version

7.6.8

### What operating system are you using?

Mac

### Did you make any manual changes to the cloud resources managed by Amplify? Please describe the changes made.

No

### Amplify Categories

auth, storage, function, api

### Amplify Commands

push

### Describe the bug

I updated amplify and made a push.
After that the app got stuck on UPDATE_ROLLBACK_FAILED and it can't be rolled back in any way.
The resource the is stuck is the User Pool, it shows the following status: 
Invalid AttributeDataType input, consider using the provided AttributeDataType enum. (Service: AWSCognitoIdentityProviderService; Status Code: 400; Error Code: InvalidParameterException; Request ID: 129f15de-7fb0-4502-847f-d83d8b4ba051; Proxy: null)

Before the push, the cli showed the following error: 
Error updating Cognito resource
Data did not validate against the supplied schema. Underlying errors were [
  {
    "keyword": "required",
    "dataPath": ".cognitoConfig",
    "schemaPath": "#/definitions/ServiceQuestionsBaseResult/required",
    "params": {
      "missingProperty": "serviceName"
    },
    "message": "should have required property 'serviceName'"
  }
]

### Expected behavior

Be able to update everything normally and have a push with no errors.

### Reproduction steps

1. Updated the amplify-cli.
2. Ran "amplify push"
3. Said yes to all the recommended updates

### GraphQL schema(s)

<details>

```graphql
# Put schemas below this line


```

</details>

### Log output

<details>

```
# Put your logs below this line


```

</details>

**Comment by luiskcs89** [comment]
I was able to make the rollback finish, but I still can't push, getting the following errors from "amplify push":

Resource handler returned message: "Invalid request provided: Queue visibility timeout: 30 seconds is less than Function timeout: 900 seconds (Service: Lambda, Status Code: 400, Request ID: 5f6e9a3f-27b1-4c82-a792-a504263dd774, Extended Request ID: null)" (RequestToken: 2ca0a0f7-ad79-1f7d-27ab-b1b8f93b38ee, HandlerErrorCode: InvalidRequest)

An error occurred during the push operation: Resource is not in the state stackUpdateComplete

**Comment by Shomari** [comment]
I'm having this exact same issue.  Happened after I added analytics. Same issue as here but doesn't seem to be fixed aws-amplify/amplify-cli#2309
